numpy/core/fromnumeric.py

```diff
@@ -1472,6 +1474,17 @@ def squeeze(a, axis=None):
     ValueError: cannot select an axis to squeeze out which has size not equal to one
     >>> np.squeeze(x, axis=2).shape
     (1, 3)
+    >>> x = np.array([[1234]])
+    >>> x.shape
+    (1, 1)
+    >>> np.squeeze(x)
+    array(1234)
+    >>> np.squeeze(x).shape
+    ()
+    >>> np.squeeze(x)[0]
+    Traceback (most recent call last):
+    ...
```

Comment:
I don't think these last two examples are useful. Perhaps add `np.squeeze(x)[()]` to replace them?